config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  post "projects/:id/contestant", to: "projects_contestant#new"
```

Comment:
the route looks good and is pointing to the correct controller the action would not be a `new` but rather a `create` because we are creating a new relationship.